pre-commit

```diff
@@ -99,3 +99,6 @@ check "IPA benchmark" \
 
 check "Sort benchmark" \
     cargo bench --bench oneshot_sort --no-default-features --features="enable-benches descriptive-gate"
+
+check "ipa-macros tests" \
+    pushd ipa-macros && cargo test --features="trybuild" && popd
```

Comment:
Is there a better way to do this? ipa-macros isn't a workspace so I can't use `cargo -p ipa-macros`.